src/Tasks/ResolveManifestFiles.cs

```diff
@@ -775,7 +782,15 @@ public void Add(ITaskItem item)
                 string fusionName = item.GetMetadata(ItemMetadataNames.fusionName);
                 if (String.IsNullOrEmpty(fusionName))
                 {
-                    fusionName = Path.GetFileNameWithoutExtension(item.ItemSpec);
+                    string destSubDir = item.GetMetadata(ItemMetadataNames.destinationSubDirectory);
+                    if (!String.IsNullOrEmpty(destSubDir))
+                    {
+                        fusionName = Path.Combine(destSubDir, Path.GetFileNameWithoutExtension(item.ItemSpec));
```

Comment:
Today, this runs on Windows only, so this is probably fine. But in the future, `Path.Combine` can return different slashes on different OSes. Anything to worry about?

Reply:
The fusionName is used as key for the entry we're writing in a dictionary so in this instance, I don't think there is an issue.

Reply:
In memory only? Great.

Reply:
Correct.